org.eclipse.january/src/org/eclipse/january/dataset/Slice.java

```diff
@@ -76,14 +105,20 @@ private Slice(final Slice other) {
 	}
 
 	@Override
+	/**
+	 * Creates a deep copy of the Slice.
+	 * @return 
```

Comment:
Remove newline/tabs between `@return` and doc (Try selecting the comment and pressing Ctrl-Shift-F to get correct formatting)